src/pages/dogs/[id].jsx

```diff
@@ -30,12 +38,47 @@ export default function IndividualDogPage() {
     }
   }, [router.query]);
 
-  if (!data || data === undefined || !data.success) {
+  useEffect(() => {
+    let logSearch = {};
+    if (logFilters) {
+      Object.keys(logFilters)
+        .filter(
+          (category) =>
+            category && Object.values(logFilters[category]).length > 0
+        )
+        .forEach((category) => {
+          logSearch[category] = Object.values(logFilters[category]);
+        });
+    }
+
+    logSearch.title = logSearchFilter;
+
+    fetch("/api/logs/search", {
```

Comment:
our /api/logs/search endpoint only filters by `dog` and 'user` log properties. other search properties like `title`, `topic`, etc are meant to be manually done on the client-side (see the slack message I sent for more info)

Reply:
fixed! As of now we create new react elements for every re-render.
I'm working on trying to store the react elements and seeing if we can toggle a visibility field, but I can't seem to figure out how to do this :/